src/Controls/src/Core.Design/AttributeTableBuilder.cs

```diff
@@ -33,6 +33,12 @@ public AttributeTableBuilder()
 			AddMemberAttributes("Microsoft.Maui.Controls.VisualElement", "FlowDirection",
 			   new TypeConverterAttribute(typeof(FlowDirectionDesignTypeConverter)));
 
+			// We need to supersede type converted declared on VisualElement.Background,
```

Comment:
```suggestion
			// We need to supersede type converter declared on VisualElement.Background,
```
not a big deal, but you know me and typos 😂